src/components/RegMenu.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  getAllRegisters,
+  getCalleeSaved,
+  getCallerSaved,
+} from './helpers/RegMenuHelpers';
+
+const RegMenu = ({ registers }) => {
+  const [filter, setFilter] = useState('all');
+
+  const CurrentCategory = () => (
+    (
+      <>
+        <button type="submit" onClick={() => setFilter('callee-saved')}>
+          callee-saved
```

Comment:
Should we capitalize the text here in the button labels? 

What about changing the name from X-saved to just X? do you think that wouldn't be descriptive enough?

Reply:
I agree with capitalized, I think keeping -saved is more descriptive, because I usually see the -saved is usually added along when referring to these kinds of registers, but if you think itll make the button look clunky, we can remove it

Reply:
yea, how it appears on the button is all that i'm worried about, since "all" is 3 characters and "caller-saved" is >10. but we can cross that bridge we get to styling the buttons